src/routing.js

```diff
@@ -119,15 +119,16 @@ export function getAppPath({
 }
 
 // Preferences
-const GLOBAL_PREFERENCES_QUERY_PARAM = '?preferences=/'
+const GLOBAL_PREFERENCES_QUERY_PARAM = '?preferences='
```

Comment:
We will need to be careful with this—I just realized that the notification service has the `/` built in and we'd have to redeploy that at the same time. Could we split this out into a different issue, and tackle it separately (as we'll either need to deploy the front and backends at similar times or be backwards compatible here)?

Reply:
Ah yes that is why i wrote this on the comment to let you know and check if it’s ok or not :)